projects/canopy/src/lib/side-nav/side-nav-bar-item/side-nav-bar-item.component.scss

```diff
@@ -0,0 +1,65 @@
+@import '../../../styles/mixins';
+
+.lg-side-nav-bar-item {
+  &:hover {
+    .lg-side-nav-bar-item__inner {
+      color: var(--color-midnight-blue);
+      cursor: pointer;
+      border-bottom: var(--side-nav-bar-item-border-width-lg) solid
+        var(--color-midnight-blue);
+      padding-bottom: var(--space-xs);
+    }
+  }
+
+  .lg-side-nav-bar-item__inner {
+    border-bottom: var(--border-width) solid var(--color-platinum);
+    color: var(--color-charcoal);
+    display: flex;
+    justify-content: space-between;
+    padding: var(--space-sm);
+    width: 100%;
+
+    @include lg-breakpoint('md') {
+      padding: var(--space-sm);
+    }
+
+    @include lg-breakpoint('lg') {
+      padding: var(--space-sm) var(--space-xs);
+    }
+  }
+
+  .lg-side-nav-bar-item__container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .lg-side-nav-bar-item__icon-container {
+    position: relative;
+    padding-right: var(--space-sm);
+
+    @include lg-breakpoint('md') {
+      padding-right: var(--space-sm);
+    }
+
+    @include lg-breakpoint('lg') {
+      padding-right: var(--space-xs);
+    }
+  }
+
+  .lg-icon {
+    position: absolute;
+    right: calc(-1 * var(--space-xs));
+  }
+}
+
+.lg-side-nav-bar-item--selected {
+  .lg-side-nav-bar-item__inner {
+    color: var(--color-super-blue);
```

Comment:
And same pattern for these variables

Reply:
oh no apologies @elenagarrone I merged just before you reviewed. I don't mind popping in to make these changes.